mkat_fpga_tests/__init__.py

```diff
@@ -17,20 +18,28 @@
 class CorrelatorFixture(object):
 
     def __init__(self, config_filename=None):
-        try:
-            subprocess.check_call(['corr2_dsim_control.py',
-            '--program', '--start'])
-            LOGGER.info('D-Eng Started succesfully')
-        except subprocess.CalledProcessError:
-            LOGGER.warn('Failed to start D-Eng')
 
         if config_filename is None:
             try:
                 config_filename = utils.parse_ini_file('/etc/corr/array0-c8n856M4k')
             except IOError:
-                LOGGER.info ("ERROR Config File Does Not Exist.")
+                LOGGER.warn ("ERROR Config File Does Not Exist.")
```

Comment:
You should only attempt to read /etc/corr/array0-c8n856M4k after calling instrument-activate on the CMC array interface.

We should have a separate config that is only for the dsim, that is independent of the correlator config.

I'll merge this now, but we should discuss and update this fixture today.